### DISCUSSION TITLE
How to display elements over the particles component?

### DISCUSSION THREAD

**Discussion by Nishant-Pall** [matteobruni/tsparticles#2186] [Q&A] [answered]
Let's say I want to display an h1 tag over the particle component, do I need to use css and z-indexes to do that or is there functionality in tsparticles-react where I can pass a component into it that should be display above it?

**Comment by matteobruni** [comment, marked as answer]
Hi @Nishant-Pall,

If you want particles in the page background you can use the `fullScreen` options, otherwise you need to use CSS to place the particles element
